README.md

```diff
@@ -76,6 +76,9 @@ const schema = new Schema('album', {
   title: { type: 'text' },
   year: { type: 'number' }
 })
+
```

Comment:
Let's make this it's own paragraph with the text "Create a repository:" and the code below.